mkosi/installer/apt.py

```diff
@@ -72,19 +66,16 @@ def apt_cmd(state: MkosiState, command: str) -> list[PathString]:
         "-o", "APT::Get::Allow-Remove-Essential=true",
         "-o", "APT::Sandbox::User=root",
         "-o", f"Dir::Cache={state.cache_dir / 'apt'}",
-        "-o", f"Dir::State={state.pkgmngr / 'var/lib/apt'}",
-        "-o", f"Dir::State::status={state.root / 'var/lib/dpkg/status'}",
-        "-o", f"Dir::Etc::trusted={trustedkeys}",
-        "-o", f"Dir::Etc::trustedparts={trustedkeys_dir}",
-        "-o", f"Dir::Log={state.pkgmngr / 'var/log/apt'}",
-        "-o", f"Dir::Bin::dpkg={shutil.which('dpkg')}",
+        "-o", f"Dir::State={state.cache_dir / 'apt'}",
```

Comment:
Could cache and state dir in the same directory collide?

Reply:
They don't seem to on my system, should be fine